### PR TITLE
[React-update] Upgrade react to 16.8.6

### PR DESCRIPTION
**Pull request by lotrien**
Upgraded React version and removed additional component (Main). It
seemed useless. Moved BrowserRouter to App component to keep all
react-router components within one file

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -11,9 +10,7 @@ const store = createStore()
 ReactDOM.render(
   (
     <Provider store={store}>
-      <BrowserRouter>
```
Comment (ikalnytskyi) [inline src/index.jsx:14]: Почему решила удалить `BrowserRouter`?

File: tests/components/Snippet.test.jsx [hunk 2]
```diff
@@ -98,20 +97,4 @@ describe('Snippet', () => {
 
     expect(spy).toHaveBeenCalled()
   })
-
-  it('should trigger copy handler on Copy button click', () => {
```
Comment (ikalnytskyi) [inline tests/components/Snippet.test.jsx:102]: Почему тест удалила?

**Review by ikalnytskyi — APPROVED** [review]

File: src/index.jsx (hunk 1 above)
Comment (lotrien) [reply src/index.jsx:14]: as I mentioned in commit message this was done to  keep all
react-router components within one file

File: tests/components/Snippet.test.jsx (hunk 2 above)
Comment (lotrien) [reply tests/components/Snippet.test.jsx:102]: it's currently fails and will be rewritten in further PRs

**Pull request merged**